packages/fxa-content-server/webpack.config.js

```diff
@@ -41,6 +41,9 @@ const webpackConfig = {
 
   resolve: {
     extensions: ['.ts', '.js', '.jsx'],
+    fallback: {
+      buffer: require.resolve('buffer/'),
```

Comment:
webpack5 does not polyfill by default.

Reply:
Ah, so the trailing slash on the package name tells it to favor the npm installed version instead of the built-in buffer package. Neato!
Per [The README](https://www.npmjs.com/package/buffer):
> _"To require this module explicitly, use `require('buffer/')` which tells the node.js module lookup algorithm (also used by browserify) to use the **npm module** named `buffer` instead of the **node.js core** module named `buffer`!"_